mix.exs

```diff
@@ -31,7 +31,9 @@ defmodule Cassette.Mixfile do
   # Type "mix help compile.app" for more information
   def application do
     [
-      extra_applications: [:logger, :sweet_xml]
+      extra_applications:
+        [:logger, :sweet_xml]
+        |> (&if(Mix.env() == :test, do: [:plug_cowboy | &1], else: &1)).()
```

Comment:
Will we only use `plug_cowboy` in dev? Or should this one be an application dependency? And what do you think about moving this one to another function?

Reply:
I was getting the warning while testing, probably because of `bypass` which is a test-only dependency

Reply:
Ah, okay!